Cargo.toml

```diff
@@ -14,20 +14,20 @@ edition = "2018"
 base64 = "0.12"
 chrono = { version = "0.4", features = ["serde"] }
 http = "0.2"
-hyper = "0.13.5"
-hyper-rustls = "0.20"
+hyper = { version = "0.14", features = ["client", "server", "tcp", "http2"] }
+hyper-rustls = { git = "https://github.com/ctz/hyper-rustls" }
```

Comment:
Is there a reason you use the git source here? I checked, and the version is the same as published, or are there minor fixes in git?

Reply:
The master branch of hyper-rustls contains an upgrade to tokio v1.0 and hyper v.014 but this is not released yet it seems.

Reply:
I see. Pinning would be better in any case, without me having done any research, is there a plan when the upgrade to tokio-1.0 will be released? I'd be more comfortable with staying within crates.io for now :-)

Reply:
Its our lucky day, after probing the author, he was kind enough to release hyper-rustls for us, you can do

`hyper-rustls = "0.22.0"`

![image](https://user-images.githubusercontent.com/2511554/103153980-4ea7f600-479d-11eb-99ba-9c1df608e079.png)

Reply:
Yes I was waiting for the released version before pinging back 😄 I've just updated it, should be good now.

Reply:
Indeed a breaking upgrade. 

hyper-rustls version was just bumped to 0.22.1 b.t.w